### PR TITLE
Created pinned articles and new KBs for auto-discover instances

### PR DESCRIPTION
**Pull request by rchakra1**

### CODE REVIEW

File: Cloud Application Manager/autodiscover-ec2-instances.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+### Auto-discover and register AWS EC2 instances in Cloud Application Manager
+
+![Unregistered Instances](../images/cloud-application-manager/aws-provider-unregisteredinstances.png)
+
+![Register Instance](../images/cloud-application-manager/aws-provider-registerinstance.png)
+
+![Register Instance Manual Intervention](../images/cloud-application-manager/aws-registerinstance-manualintervention.png)
+
+![Agent Install](../images/cloud-application-manager/aws-provider-agentinstallscript.png)
```
Comment (srveit) [inline Cloud Application Manager/autodiscover-ec2-instances.md]: Change link to `../images/cloud-application-manager/aws-provider-agentInstallscript.png`. Notice the capital "I" in "Install".

**Comment by rchakra1** [comment]
@srveit good catch. thanks for pointing that out. I fixed and pushed it. Do you want me to submit a separate PR?

File: Cloud Application Manager/autodiscover-ec2-instances.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+### If you have an existing AWS provider in Cloud Application Manager
+
+The next time you click on sync, Cloud Application Manager will auto-discover those instances that exist in AWS EC2 but have not been provisioned using Cloud Application Manager and save them in the Unregistered instances tab under the Provider details. You can follow the on-screen instructions to register them in Cloud Application Manager.
+
+To register AWS EC2 instance, an additional step is required. Cloud Application Manager uses UserData to install the Cloud Application Manager agent on provision time. Since the instance was initially provisioned outside of Cloud Application Manager, users have to execute a script to install the Cloud Application Manager agent.
+
+
+### Auto-discover and register AWS EC2 instances in Cloud Application Manager
+
+![Unregistered Instances](../images/cloud-application-manager/aws-provider-unregisteredinstances.png)
```
Comment (srveit) [inline Cloud Application Manager/autodiscover-ec2-instances.md]: Change link to `../images/cloud-application-manager/aws-provider-unregisteredInstances.png`

File: Cloud Application Manager/autodiscover-ec2-instances.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+![Register Instance](../images/cloud-application-manager/aws-provider-registerinstance.png)
```
Comment (srveit) [inline Cloud Application Manager/autodiscover-ec2-instances.md]: Change link to `../images/cloud-application-manager/aws-provider-registerInstance.png`

**Comment by srveit** [comment]
@rchakra1 You do not seed to submit a separate PR. Make the changes on your branch and re-sync.

**Comment by rchakra1** [comment]
@srveit Fixed the suggested changes

**Pull request merged**